test/integration/processing/payment-schedule/get-payment-schedule.test.js

```diff
@@ -116,116 +116,128 @@ describe('get payment schedule', () => {
     expect(result.sbi).toEqual(paymentSchedule.sbi)
   })
 
-  test('returns schedule key', async () => {
+  test('returns remainingAmount key', async () => {
```

Comment:
You need to make copies of these test for the remainingAmount key, not just overwrite those for schedule.

Schedule is still outputted as part of this function.

Reply:
Did cut and paste instead of copy and paste

Reply:
I actually did copy and paste, it is in line 129

Reply:
Yup, thanks GIIIIITT!